src/pip/_internal/cli/cmdoptions.py

```diff
@@ -399,30 +397,26 @@ def editable():
 )  # type: Any
 
 
-def _get_format_control(values, option):
```

Comment:
Instead of removing this, you can probably use it in _handle_no_binary and _handle_only_binary.